airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -299,7 +299,7 @@ protected DataSource createDataSource(final JsonNode config) {
         jdbcConfig.has(JdbcUtils.PASSWORD_KEY) ? jdbcConfig.get(JdbcUtils.PASSWORD_KEY).asText() : null,
         driverClass,
         jdbcConfig.get(JdbcUtils.JDBC_URL_KEY).asText(),
-        JdbcUtils.parseJdbcParameters(jdbcConfig, JdbcUtils.CONNECTION_PROPERTIES_KEY, getJdbcParameterDelimiter()));
+        getConnectionProperties(config));
```

Comment:
doublechecking that I have the right context: the JdbcUtils method doesn't verify invalid parameter overrides, but getConnectionProperties does that verification (but otherwise behaves identically)?

Reply:
Correct, `getConnectionProperties` is superset that includes the same method `JdbcUtils#parseJdbcParameters`, there is a slight modification where the `key` (2nd parameter) that it is parsed on is `JdbcUtils.JDBC_URL_PARAMS_KEY` vs `JdbcUtils.CONNECTION_PROPERTIES_KEY` which is to have parity for `AbstractJdbcDestination#getConnectionProperties`

Interestingly the Map of values for `JdbcUtils.CONNECTION_PROPERTIES_KEY` is actually created from the same parameters that is generated for `JdbcUtils.JDBC_URL_PARAMS_KEY`. There seems to be overlap that isn't clear _why_ there are two keys that are generated from the same values but to keep scope we're matching parity first then will file another ticket to address the cruft with overlapping `JdbcUrl` params

The `getConnectionProperties` was merged previously in the original commit on `AbstractJdbcSource#329`